README.adoc

```diff
@@ -1092,7 +1092,7 @@ elem # => NameError: undefined local variable or method `elem'
 
 === No `then` [[no-then]]
 
-Do not use `then` for multi-line `if`/`unless`.
+Do not use `then` for multi-line `if`/`unless` or `when`.
```

Comment:
It seems that `/` is conventionally used as a separator.

```diff
- Do not use `then` for multi-line `if`/`unless` or `when`.
+ Do not use `then` for multi-line `if`/`unless`/`when`.
```

Reply:
True, I'll fix it.